docs/content/quickstart/build-real-time-data-lake-tutorial.md

```diff
@@ -0,0 +1,257 @@
+We can also visit [http://localhost:8081/](http://localhost:8081/) to see if Flink is running normally.
+![Flink UI](/_static/fig/real-time-data-lake-tutorial/flink-ui.png "Flink UI")
+
+### Preparing data in databases
+1. Enter mysql's container:
+    ```shell
+    docker-compose exec mysql mysql -uroot -p123456
+    ```
+2. Create databases/tables and populate data:
+
+   Create a logical sharding table `user` sharded in different databases and tables physically, and one of them misses a column.
+   ```sql
+    CREATE DATABASE user_db_1;
+    USE user_db_1;
+    CREATE TABLE user_table_1 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+    INSERT INTO user_table_1 VALUES (11100,"user_11100","Shanghai","123567891234","user_11100@foo.com");
+   
+    CREATE TABLE user_table_2 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+   INSERT INTO user_table_2 VALUES (12100,"user_12100","Shanghai","123567891234","user_12100@foo.com");
+   ```
+   ```sql
+   CREATE DATABASE user_db_2;
+   USE user_db_2;
+   CREATE TABLE user_table_1 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512)
+   );
+   INSERT INTO user_table_1 VALUES (21100,"user_21100","Shanghai","123567891234");
+
+   CREATE TABLE user_table_2 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512),
+     email VARCHAR(255)
+   );
+   INSERT INTO user_table_2 VALUES (22100,"user_22100","Shanghai","123567891234","user_22100@foo.com");
+   ```
+
+## Creating tables using Flink DDL in Flink SQL CLI
+First, use the following command to enter the Flink SQL CLI Container:
+```shell
+docker-compose exec sql-client ./sql-client
+```
+
+We should see the welcome screen of the CLI client:
+![Flink SQL Client](/_static/fig/real-time-data-lake-tutorial/flink-sql-client.png  "Flink SQL Client" )
+
+Then do the following steps in Flink SQL CLI:
+
+1. Enable checkpoints every 3 seconds
+   ```sql
+   -- Flink SQL                   
+   Flink SQL> SET execution.checkpointing.interval = 3s;
+   ```
+2. Create source table 
+
+   Create a source table that captures the data from the logical sharding table `user`. Here, we use regex to match all the physical tables.
+   The table `user_source` contains all the columns, and the column's value will be null if the record from the underlying table misses the column.
+   Also, the table defines metadata column to identify which database/table the record comes from.
+   ```sql
+   -- Flink SQL
+   Flink SQL> CREATE TABLE user_source (
+       db_name STRING METADATA FROM 'database_name' VIRTUAL,
```

Comment:
```suggestion
       database_name STRING METADATA VIRTUAL,
```

As a demo, we don't need to show all possible abilities, but show the simplest way and keep consistent in demo. For advantage usages, they can refer the official documentation.